api/ClipboardItem.json

```diff
@@ -67,10 +67,14 @@
           "description": "<code>ClipboardItem()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "66"
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data. Full implementation would also allow for a string or a Promise which resolves with either a string or blob. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
             },
             "chrome_android": {
-              "version_added": "66"
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data. Full implementation would also allow for a string or a Promise which resolves with either a string or blob. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
             },
             "edge": {
               "version_added": "≤79"
```

Comment:
The note on Chrome should probably be carried over to Edge, Opera, and Samsung Internet, too